types/event_bus.go

```diff
@@ -153,19 +163,14 @@ func (b *EventBus) PublishEventTx(data EventDataTx) error {
 	// no explicit deadline for publishing events
 	ctx := context.Background()
 
-	tags := b.validateAndStringifyTags(data.Result.Tags, b.Logger.With("tx", data.Tx))
+	events := b.validateAndStringifyEvents(data.Result.Events, b.Logger.With("tx", data.Tx))
 
 	// add predefined tags
-	logIfTagExists(EventTypeKey, tags, b.Logger)
```

Comment:
same here. this method allows user to see if some tag will be overwritten by Tendermint

Reply:
See response [here](https://github.com/tendermint/tendermint/pull/3643#discussion_r291264222).